restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/ConferencesEndpoint.java

```diff
@@ -194,4 +202,75 @@ protected Response getConferences(final String accountSid, UriInfo info, MediaTy
         }
     }
 
+    protected Response updateConference(String accountSid, String sid, MultivaluedMap<String, String> data,
+            MediaType responseType) {
+        logger.info(String.format("updateConference accountsid: %s conferenceSid: %s", accountSid, sid));
+        Account account = daoManager.getAccountsDao().getAccount(accountSid);
+        Account effectiveAccount = userIdentityContext.getEffectiveAccount();
+        try {
+            secure(account, "RestComm:Modify:Conferences");
```

Comment:
@maria-farooq Since later we use `isSuperAdmin()` I don't think we need to check permissions here.
The `updateConference()` method could be protected with just the `isSuperAdmin()`

Reply:
conference termination request can be initiated by owner of the conference as well hence we need to make sure that resource is being accessed by proper account